modules/junit-jupiter/src/main/java/org/testcontainers/junit/jupiter/TestcontainersTestDescription.java

```diff
@@ -0,0 +1,20 @@
+package org.testcontainers.junit.jupiter;
+
+import lombok.RequiredArgsConstructor;
+import org.testcontainers.lifecycle.TestDescription;
+
+@RequiredArgsConstructor
```

Comment:
```suggestion
@Value
```

so that you can remove `private final` and the getters

Reply:
@bsideup Since lombok dose not make the fields private it's "ok" that they're package private?

Reply:
AFAIK `@Value` does make them private:
https://projectlombok.org/features/Value